assembl/static2/css/components/text.scss

```diff
@@ -110,6 +110,14 @@
   font-family: $font-family4;
 }
 
+.light-title-7 {
```

Comment:
Title 6 and 7 are not shown in the `styleguide`, especially in these formats. @paolina92 and I can help you edit the styleguide so that everyone can visually see what these changes that are being proposed looks like.

Reply:
Also, if we are getting to 7 different light title styles, isn't there a problem of unity in the UI?